### ISSUE TITLE
Underline hyperlinks in text

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#1152]
From Todd:

We would very much like to underline our text hyperlinks in SiteNow as a stylistic means of distinguishing them from the surrounding text. It seems like this should at least be an option (probably the default), as this distinction makes it easy to scan a page and pick out the links. It’s also a de facto standard and people are used to it. Finally, it’s a sure thing that every link will fail every a11y test going forward.
 
I see that the style rule for <a> is actually set to remove the text decoration. It’s hard to understand the complete elimination of a common and standard behavior in favor of one that effectively reduces a11y and usability. Is there any way we can restore the text-decoration, or at least leave it up to the page author?
 
I know there are additional themes in the works, but this feels like something we can address in real time. I look forward hearing your thoughts.

**Comment by briand44** [comment]
I think this depends on how long the uiowa_bootstrap theme is going to stay in place.

**Comment by joewhitsitt** [comment]
This is affecting the footer contact info block links. It is a text area that can contain links and text:

```
It appears that color is the only distinguishing feature about links in blocks of text.

If links in blocks of text are identified only by color, the color contrast ratio between the link text and the surrounding text needs to be at least 3:1. This is not the case here.

Furthermore, for links identified only by color there should be additional visual cues when users point to the link with their mouse or move keyboard focus to the link. The additional visual cues could be underlining the link or making it bold.

This applies to links that are visually part of a block of text. Menu items etc. should not be considered here.

Choose one of the following:

Make sure there are other visual indicators of links besides the color, such as underline or bold.

Or ensure that links have a contrast ratio of at least 3:1 with surrounding text, and provide additional visual cues when pointing with a mouse or moving keyboard focus to the link.

Please notice that the 3:1 color contrast ratio for links identified only by color should be measured text to text and not text to background.
```

**Comment by joewhitsitt** [comment]
**uids_base body example**

<img width="1120" alt="Screen Shot 2020-07-08 at 4 46 18 PM" src="https://user-images.githubusercontent.com/4663676/86973667-96c55480-c13a-11ea-8755-9a9feef13387.png">

So, uiowa_bootstrap is going away and uids_base incorporates underlined links in main content areas that aren't otherwise buttons.

Additionally, we are resolving SiteImprove flags related to https://www.w3.org/TR/WCAG20-TECHS/F73 and https://www.w3.org/TR/WCAG20-TECHS/G183 as we identify them.

We are addressing initial specific SiteImprove a11y issues here: #1775 . Closing this issue in favor of that.
